### ISSUE TITLE
NetEase - Remove Lyric Lines with "作词" 

### ISSUE THREAD

**Issue by ngs428** [jacquesh/foo_openlyrics#423]
Lyrics from NetEase will contain Chinese characters at the beginning of the first 1 or 2 synchronized lines.  The characters are show as below:

[00:00.00]作词 : Don Henley/J.D. Souther/Danny Kortchmar
[00:00.78]作曲 : Don Henley/J.D. Souther/Danny Kortchmar

Possible to have an Auto-Edit to remove these lines?

**Comment by jacquesh** [comment]
I think if I do this, it'd be as an optional modification, configured to either always or never happen automatically for everything retrieved from NetEase

**Comment by ngs428** [comment]
Yes, that would make sense.  I end up manually deleting them each time, so was hoping you’d have a better way.  Thank you!

**Comment by TapioOfTapiola** [comment]
Haha, I suggested it https://github.com/jacquesh/foo_openlyrics/issues/426 and then noticed this thread. Yeah, Netease sucks in that regard, I had to remove it for the auto taggings.
